### ISSUE TITLE
Issues when translating content with a HTML tag

### ISSUE THREAD

**Issue by guillelopezgines** [while1618/i18n-auto-translation#1]
Hi, let me thank you first for developing this translation package.

I just started using the package, it works great, except when finding some content  that includes HTML tags:

```
{
  "general": {
    "404": {
      "title_html": "404 Page <br> not found",
      "subtitle": "The page you requested doesn't exist.",
      "continue_shopping_html": "Continue shopping",
      "open_menu": "Open menu"
    }
  }
}
```

It gets translated to:

```
{
  "general": {
    "404": {
      "title_html": "404 página<br> not found La página solicitada no existe. Seguir comprando Abrir menú"
    }
  }
}
```
I am using the "google-official" provider.

**Comment by while1618** [comment]
Hi, thanks for the feedback, I just checked it, and yes, you are right, there is a problem when translating with "google-official".

I also checked with other providers, and they are working as expected, so it's a problem just with google. I'll try to find a solution, and I'll inform you here.

**Comment by while1618** [comment]
It's fixed. Update i18n-auto-translation to 1.1.7 and try to run it again.

**Comment by guillelopezgines** [comment]
Thanks @while1618 for the quick fix!

It's working, but I am seeing something that might be easy to fix: 

When adding an HTML tag some characters change the encoding.

For example, with no HTML tag:

```
{
  "general": {
    "newsletter_form": {
      "no_results_for": "Your search for lorem did not yield any results:"
    }
  }
}
```
Translates to:
```
{
  "general": {
    "newsletter_form": {
      "no_results_for": "Votre recherche de lorem n'a donné aucun résultat :"
    }
  }
}
```

But when it has HTML tag:
```
{
  "general": {
    "newsletter_form": {
      "no_results_for": "Your search for <b>lorem</b> did not yield any results:"
    }
  }
}
```
Then it goes:
```
{
  "general": {
    "newsletter_form": {
      "no_results_for": "Votre recherche de <b>lorem</b> n&#39;a donné aucun résultat :"
    }
  }
}
```
Notice the difference between `n'a` and `n&#39;a`

**Comment by while1618** [comment]
Fixed in v1.1.8
